core/src/navigation/TopNav.svelte

```diff
@@ -704,6 +302,522 @@
+                                    >{getNodeLabel(node)}</span
+                                  >
+                                </a>
+                              </li>
+                            {:else}
+                              <li class="fd-menu__item">
+                                <a
+                                  href={getRouteLink(node)}
+                                  title={resolveTooltipText(
+                                    node,
+                                    getNodeLabel(node)
+                                  )}
+                                  class="fd-menu__link"
+                                  on:click|preventDefault={() =>
+                                    openMobileTopNavDropDown(node)}
+                                  data-e2e="mobile-topnav-dropdown-category"
+                                >
+                                  <span class="fd-top-nav__icon">
+                                    {#if hasOpenUIicon(node)}
+                                      <i
+                                        class="sap-icon {node.icon &&
+                                        hasOpenUIicon(node)
+                                          ? getSapIconStr(node.icon)
+                                          : ''}"
+                                      />
+                                    {:else}
+                                      <img
+                                        src={node.icon}
+                                        alt={node.altText ? node.altText : ''}
+                                      />
+                                    {/if}
+                                    <BadgeCounter {node} />
+                                  </span>
+                                  <span class="fd-list__title"
+                                    >{getNodeLabel(node)}</span
+                                  >
+                                </a>
+                              </li>
+                              <!-- {:else}
+                <li class="fd-menu__item">
+                  <a
+                    href="{getRouteLink(node)}"
+                    title="{getNodeLabel(node)}"
+                    class="fd-menu__link"
+                    on:click|preventDefault="openMobileTopNavDropDown(node)"
+                    data-testid="mobile-topnav-dropdown-category"
+                  >
+                    {#if node.icon} {#if hasOpenUIicon(node)}
+                    <span
+                      class="fd-top-nav__icon sap-icon {getSapIconStr(node.icon)}"
+                    ></span>
+                    {:else}
+                    <img
+                      class="fd-top-nav__icon nav-icon"
+                      src="{node.icon}"
+                    >
+                    {/if} {/if}
+                    <span class="fd-list__title">{getNodeLabel(node)}</span>
+                  </a>
+                </li>-->
+                              <!-- {/if} -->
```

Comment:
It seems no need to keep this part. Please remove it.

Reply:
above code does this, so I removed this commented code